scipy/stats/mstats_basic.py

```diff
@@ -567,6 +586,7 @@ def _kendall_p_exact(n, c):
             if j <= c:
                 new[j:] -= new[:c+1-j]
         prob = 2.0*np.sum(new)/math.factorial(n)
+        p_mass_at_c = new[-1]/math.factorial(n)
```

Comment:
`new` appears to calculate the frequency (under all permutations) of each possible value of the test statistic from `0` to `c`, so the last element is the number of permutations for which we'd get the observed value of `c`.